src/app/character-tile/StoreHeading.scss

```diff
@@ -22,7 +23,7 @@
     top: 0;
     left: 0;
     z-index: 1;
-    border-top: 13px solid $orange;
+    border-top: 13px solid var(--theme-accent-primary);
```

Comment:
Since this denotes the user's active character, perhaps this should be [the green that we use for the refresh button "is playing" indicator instead](https://github.com/DestinyItemManager/DIM/blob/4be7854f229d76108e3ebbff41545bc6f9abeaf4/src/app/shell/RefreshButton.m.scss#L10-L12)?

Reply:
This one's been around long enough that I'd probably leave it as the orange.